File: Java Script - Introduction to Programming/4.1. Complex Conditions/10. Animal Type.js
function animalType([animal]) {
  let type = '';
  switch (animal) {
    case 'dog': type = 'mammal'; break;
    case 'crocodile': type = 'reptile';
    case 'tortoise': type = 'reptile';
    case 'snake': type = 'reptile'; break;
    default: return 'unknown'
  }

 return type
}

//console.log(animalType(['snake']))

File: Pyrhon - Introduction to Programming/4.1. Complex Conditions/10. Animal Type.py
def animal_type(animal):
    switcher = {
        'dog': 'mammal',
        'crocodile': 'reptile',
        'tortoise': 'reptile',
        'snake': 'reptile'
    }

    return switcher.get(animal ,'unknown')


print(animal_type(input()))


'''
snake
#reptile 
'''
